release/train_tests/e2e/test_persistence.py

```diff
@@ -312,6 +314,16 @@ def test_trainer(storage_path_storage_filesystem_label, tmp_path, monkeypatch):
     print(aggregated_metrics)
     update_output_json(flatten_dict({label: aggregated_metrics}))
 
+    print("Deleting files from the run...")
```

Comment:
Another follow up - wondering if we have some sort of deletion policy set for these files, so that they get cleaned up even if the test fails before and doesn't get to this step.

Reply:
`ANYSCALE_ARTIFACT_STORAGE` is treated as a customer storage on anyscale. I do not think it has any retention (and if it has, people that uses it will kind of go mad when they see that their model weights are lost..)

Reply:
you can configure manual retention policy on certain prefix though.